### PR TITLE
Update xtermjs and socketio

### PR DESCRIPTION
**Pull request by manics**
Thanks for creating this project as a learning resource. I thought I'd update the dependencies:
- xterm 4.11.0
- socket.io 4.0.1
- flask-socketio 5

### CODE REVIEW

File: pyxtermjs/index.html [hunk 1]
```diff
@@ -20,31 +20,32 @@
   built by <a href="https://grassfedcode.com">Chad Smith</a> <a href="https://github.com/cs01">GitHub</a>
 </p>
 <!-- xterm -->
-<script src="https://unpkg.com/xterm@3.6.0/dist/xterm.js"></script>
-<script src="https://unpkg.com/xterm@3.6.0/dist/addons/fit/fit.js"></script>
-<script src="https://unpkg.com/xterm@3.6.0/dist/addons/webLinks/webLinks.js"></script>
-<script src="https://unpkg.com/xterm@3.6.0/dist/addons/fullscreen/fullscreen.js"></script>
```
Comment (manics) [inline pyxtermjs/index.html:26]: The fullscreen addon is no longer supported

**Comment by cs01** [comment]
Thanks for this! It will definitely be good to keep the "bit rot" off this as much as possible.

I tried it out with
```
python3.7 -m venv venv
source venv/bin/activate
pip install -r requirements.txt 
python -m pyxtermjs
```
but in the browser, the state remains on "connecting" and the terminal doesn't work. Could you provide steps to reproduce it working?

**Review by cs01 — CHANGES_REQUESTED** [review]
Could not get this branch to run in my computer

**Comment by manics** [comment]
Sorry for the delay in responding. I've tested this again just now:
```
conda create -n tmp python=3.7
conda activate tmp
pip install -r requirements.txt 
pip install .
python -m pyxtermjs
```
http://127.0.0.1:5000/ loads successfully. I've tried this in Firefox 88.0 and Chrome 90.0.4430.93 on Fedora Linux 33.

**Comment by cs01** [comment]
> Sorry for the delay in responding.

I too am sorry for my delayed reply :). I tried out your change again, and it worked. I think the fix is to update `setup.py` to install from requirements.txt. I'll merge your change as-is and then fix setup.py some more.

**Pull request merged**